Source/RpmAvatarCreator/Public/NARF/Buttons/RpmAssetButton.h

```diff
@@ -0,0 +1,35 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "RpmButtonBase.h"
+#include "RpmAvatarCreatorTypes.h"
+#include "RpmAssetButton.generated.h"
+
+
+DECLARE_DYNAMIC_MULTICAST_DELEGATE_OneParam(FOnAssetSelected, FRpmPartnerAsset, PartnerAsset);
+
+UCLASS(Blueprintable)
+class RPMAVATARCREATOR_API URpmAssetButton : public URpmButtonBase
```

Comment:
There is already an asset button class in the UI folder why did you introduce a new one?